docs/_docs/getting-started/getting-started.md

```diff
@@ -4,15 +4,24 @@ excerpt: ""
 layout: docs
 permalink: /docs/getting-started/
 ---
+
+## 1. Create a Free Account at codenvy.io
+Everyone gets 3GB for free. Connect to a public toolchain and be coding in minutes.
+
+## 2. Download Codenvy
+Install Codenvy anywhere Docker runs and connect it to your private toolchain. Control security and configuration–even add your own extensions.
+
 With Docker 1.11+ on Windows, Mac, or Linux:
 ```
 $ docker run codenvy/cli start
 ```
 This command will give you additional instructions on how to run the Codenvy CLI while setting your hostname, configuring volume mounts, and testing your Docker setup.
 
+
+## Where To Go Next
 | I am a...   | Consider going to... |
 | --- | --- |
 | **Developer** who will be using Codenvy for my projects. | [*The User Guide*](../../docs/workspace-admin-intro/). This will teach you how to configure and use workspaces so that they bend to your magical programming will. |
-| **Developer** who will be creating custom extensions, plug-ins, and assemblies to package Codenvy and Eclipse Che into new tools. | [*The Developer Guide*](../../docs/rest-api). These documents provide all of our API interfaces, both internal and external. You will need to get comfortable with using the Eclipse Che SDK, and there is extensive additional SDK documentation that will make you an innovation beast. Also, if you are going to place your extensions into a Codenvy On-Prem, check out the additional [Plug-Ins](../../docs/developing-extensions/) administrator steps. |
+| **Developer** who will be creating custom extensions, plug-ins, and assemblies to package Codenvy and Eclipse Che into new tools. | [*Che Plugin Guide*](https://eclipse-che.readme.io/docs/introduction-1). These docs will help you setup your dev evironment and walk you through how to craete server- and client-side extensions and plugins. If you are going to place your extensions into a Codenvy On-Prem instance please [contact us for guidance](https://codenvy.com/contact/questions/). |
```

Comment:
I'll fix the typo.

Although it's cool that you can see the page, it's confusing not to have it in the TOC and I can't jump to the other pages in that section so jumping away to the Che docs site is better. We can update the link after with the correct github.io link.